task_sdk/src/airflow/sdk/definitions/asset/__init__.py

```diff
@@ -391,10 +391,10 @@ class Model(Asset):
 
 @attrs.define(unsafe_hash=False)
 class AssetAlias(BaseAsset):
-    """A represeation of asset alias which is used to create asset during the runtime."""
+    """A representation of asset alias which is used to create asset during the runtime."""
 
     name: str = attrs.field(validator=_validate_non_empty_identifier)
-    group: str = attrs.field(kw_only=True, default="", validator=_validate_identifier)
+    group: str = attrs.field(kw_only=True, default="asset", validator=_validate_identifier)
```

Comment:
Not sure whether it should be `asset` or `asset-alis`. Either way I'm ok. just want to point it out

Reply:
Yeah or it could be just `alias`. I have no idea.